manifests/init.pp

```diff
@@ -347,6 +351,13 @@
   if $keys {
     create_resources('apt::key', $keys)
   }
+  if $keyrings {
+    $keyrings.each |$key, $data| {
+      apt::keyring { $key:
+        * => $data,
+      }
+    }
+  }
```

Comment:
The parameter has a default value and will never be `undef`, so we can remove the condition:

```suggestion
  $keyrings.each |$key, $data| {
    apt::keyring { $key:
      * => $data,
    }
  }
```

Reply:
Sure, Thanks. Done